### DISCUSSION TITLE
Code Block is not an actual 'code block'

### DISCUSSION THREAD

**Discussion by kobaz** [requarks/wiki#5827] [Error / Bug Report] [answered]
When inserting a 'Code Block' in the visual editor you get a strangely formatted double-spaced text area like this:
(Screenshot from wikijs visual editor, 2.5.289)
![image](https://user-images.githubusercontent.com/1695852/200575136-696f571c-20c3-47e8-8fbc-3ba00e5b6a5c.png)

When one would expect a code block that looks like a traditional codeblock that's ubiquitous around the internet that looks like this:
(Screenshot from Atlassian Confluence)
![image](https://user-images.githubusercontent.com/1695852/200575252-2409999b-f205-47de-a051-0b420f380ef0.png)

Even a github block here below, is a good example of what a code block should render as
(Github code block using `)
```
asdf
asdf
asdf
```

**Comment by NGPixel** [comment, marked as answer]
As answered on Slack, you need to use **code blocks**, not **inline code**.

_(posting here in case someone finds this question later on)_

**Comment by kobaz** [comment]
Hey, sorry for the crosspost I forgot to cancel my request here when I got an answer.

Thanks!
